drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/builders/ParameterUtilities.java

```diff
@@ -60,12 +80,20 @@ public String convertSingleParameterToTemplateKey(String xlsTemplate) {
         }
         parameterCounter++;
         matcherSingleParameter.appendTail( result );
+
+        if ( part == ParameterizedValueBuilder.Part.LHS ) {
```

Comment:
This block seems to be the same in ```convertIndexedParametersToTemplateKeys``` and ```convertSingleParameterToTemplateKey```. Prob. better to create a method and reuse?